src/System.Collections/src/System/Collections/Generic/SortedDictionary.cs

```diff
@@ -11,6 +11,7 @@ namespace System.Collections.Generic
     [DebuggerTypeProxy(typeof(IDictionaryDebugView<,>))]
     [DebuggerDisplay("Count = {Count}")]
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
I disabled the SortedDictionary<,> test for now (until we have a decision) as I would have to update the blob for it (because the binary format changed after commenting the TypeForwardedFrom from TreeSet<,>).